index.js

```diff
@@ -514,7 +514,7 @@ function launchApp(argo, startExternalAdapterServer) {
         if (openfinWinOpts && !isRunning) {
             //making sure that if a window is pressent we set the window name === to the uuid as per 5.0
             openfinWinOpts.name = uuid;
```

Comment:
This statement still succeeds only because of the nature of `getWindowOptions(...)` (terribly named function, I might add, if you look at what the function actually does).

It is equivalent to:

```
configObject.startup_app.name = uuid;
```

As written, the code is not exactly clear / maintainable, because it requires understanding the implementation details of `getWindowOptions(...)` function.

Also, it is brittle code: if the `getWindowOptions` call were ever refactored to return a copy of the object, for example, the assignment would fail.

**Suggestion**: If setting the name = to UUID is explicitly for 5.0 compatibility, doesn't that behavior belong in the `convertOptions` utility? Isn't that its entire purpose?